rest/src/main/groovy/whelk/rest/api/SearchUtils.groovy

```diff
@@ -221,11 +224,27 @@ class SearchUtils {
                                            items, mappings, pageParams,
                                            limit, offset, total)
 
-        if (stats && !suggest) {
+        if (stats && (!suggest || spell != "only")) {
             stats[JsonLd.ID_KEY] = '#stats'
             result['stats'] = stats
         }
 
+        if (spell) {
+            result['_spell'] = []
+            esResult["spell"].each { Map suggestion ->
+                result['_spell'] << [
+                        'label': suggestion['text'],
+                        'labelWithMarkup': suggestion['highlighted'],
```

Comment:
```suggestion
                        'labelHTML': suggestion['highlighted'],
```

Found this usage of type coercion for description with type `rdf:HTML`.
We can stick to the same naming convention.

https://github.com/libris/definitions/blob/a320ff90c87a96e143826812a50afb0635c8cccd/source/app/help.jsonld#L12